index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="body-wrapper">
+        <nav class="header-flex">
+            <a class="logo" href=""><h5>Omar Mohammad</h5></a><span>
+            <img class="union" src="./assets/Union.png" alt='union-sign'/></span>
+        </nav>
+        <div class="container-flex">
+            <div class="intro-flex">
+                <h1>Omar Mohammad</h1>  
+                <h2>Software Developer</h2>        
+            </div>
+        <br>
+            <div class="about-flex">
+                <p class="about"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
```

Comment:
- [ ] the same thing here, please change the font color for this paragraph to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)

> the font color should be `color: #C8C8D4;`